.devcontainer/.env.template

```
@@ -83,3 +83,10 @@ ZMS_API_PASSWORD_CITIZENAPI=vorschau
 # HTTPS_PROXY=http://proxy.example.com:80
 # https_proxy=http://proxy.example.com:80
 # NO_PROXY=localhost,127.0.0.1,keycloak,zms-web,zms-db
+
+# zmscitizenapi: if HTTP_HOST or X-Forwarded-Host contains this substring, unpublished
+# services/relations are still returned (e.g. Munich SADB ZMS_INTERN=true → public=false after import).
+# One substring per env; comma-separated lists are not supported by the app.
+# Do not use zms-web here unless you never call the citizen API on your public ddev URL (it would stop matching).
+# refarch-gateway citizen-api uses PreserveHostHeader so the browser host is forwarded to PHP.
+ACCESS_UNPUBLISHED_ON_DOMAIN=localhost
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)